src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,36 @@
-export const TodoList = () => {};
+import { User } from '../../Types/User';
+import { Todo } from '../../Types/Todo';
+import { TodoInfo } from '../TodoInfo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todos: Todo[],
+  users: User[],
+};
+
+const TodoList: React.FC<Props> = ({ todos, users }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((todo) => (
+        <article
+          data-id={todo.id}
+          className={
+            todo.completed ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'
+          }
+          key={todo.id}
+        >
+          <TodoInfo todo={todo} />
+          {users
+            .filter((user) => todo.userId === user.id)
+            .map((user) => (
+              <div key={user.id}>
+                <UserInfo user={user} />
+              </div>
```

Comment:
You should prepare the data before. Add user property to each todo
![image](https://github.com/mate-academy/react_add-todo-form/assets/96409563/058c4780-49f2-4b03-9889-a649a712ecbf)